CHANGELOG.md

```diff
@@ -17,13 +17,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   from `ContainerFuture` to `CivisFuture` (#426)
 - Updated the docstrings for `file_to_civis` (for `buf` and `expires_at`),
   `dataframe_to_file` (for `expires_at`), and `json_to_file` (for `expires_at`). (#427)
-- Added default values from swagger in client method's signature (#416)
```

Comment:
Just cleaning up -- this is a duplicate entry for the same one a couple lines above